boot/bootstate20.go

```diff
@@ -95,6 +117,9 @@ type bootStateUpdate20 struct {
 
 	// tasks to run after the modeenv has been written
 	postModeenvTasks []bootCommitTask
+
+	// resealNeeded set if we need to reseal keys
+	resealNeeded bool
```

Comment:
this is not used anymore, should be removed

Reply:
Done now